runtime/context.go

```diff
@@ -319,12 +321,16 @@ func NewContextMetrics(scope tally.Scope) ContextMetrics {
 
 // IncCounter increments the counter with current tags from context
 func (c *contextMetrics) IncCounter(ctx context.Context, name string, value int64) {
-	tags := GetScopeTagsFromCtx(ctx)
-	c.scope.Tagged(tags).Counter(name).Inc(value)
+	c.scope.Tagged(GetScopeTagsFromCtx(ctx)).Counter(name).Inc(value)
 }
 
+// @deprecated
 // RecordTimer records the duration with current tags from context
-func (c *contextMetrics) RecordTimer(ctx context.Context, name string, d time.Duration) {
-	tags := GetScopeTagsFromCtx(ctx)
-	c.scope.Tagged(tags).Timer(name).Record(d)
+//func (c *contextMetrics) RecordTimer(ctx context.Context, name string, d time.Duration) {
+//	c.scope.Tagged(GetScopeTagsFromCtx(ctx)).Timer(name).Record(d)
```

Comment:
Not crazy about all the commented code. Can we just deleted all of these.

Reply:
Done.